### DISCUSSION TITLE
Add data from IIT Kanpur source

### DISCUSSION THREAD

**Discussion by shuvashish76** [WirelessAlien/BhagavadGitaApp#4] [Ideas]
Would be nice if we could fetch Audio, Translations & Commentaries from https://www.gitasupersite.iitk.ac.in/ which is a nice reliable source.
I'm not sure about their license, maybe try to contact them for more info.

**Comment by WirelessAlien** [comment]
> Would be nice if we could fetch Audio, Translations & Commentaries from https://www.gitasupersite.iitk.ac.in/ which is a nice reliable source.

Yeah. It would be excellent 👌. 
> I'm not sure about their license, maybe try to contact them for more info.

I actually sent a message to them yesterday. They will reply, I hope.
